CMakeLists.txt

```diff
@@ -73,12 +78,39 @@ SET_LIBRARY_TARGET_PROPERTIES("gstcef")
 add_dependencies("gstcef" libcef_dll_wrapper)
 target_link_libraries("gstcef" libcef_lib libcef_dll_wrapper ${CEF_STANDARD_LIBS} ${GST_LIBRARIES})
 target_include_directories("gstcef" PUBLIC ${GST_INCLUDE_DIRS})
-target_compile_definitions("gstcef" PUBLIC "-DCEF_SUBPROCESS_PATH=\"$<TARGET_FILE:gstcefsubprocess>\"")
-target_compile_definitions("gstcef" PUBLIC "-DCEF_LOCALES_DIR=\"${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/locales\"")
+
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  set(CEF_SUBPROCESS_BASE_PATH "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}")
+else()
+  set(CEF_SUBPROCESS_BASE_PATH "${CMAKE_INSTALL_PREFIX}")
```

Comment:
I think this means cefsrc won't be usable "uninstalled" when install prefix doesn't it?

Reply:
yes. The thing with absolute path is that it will work either installed or uninstalled but never both. The only thing I can think of that could allow for both ways of usage would be to get the path somehow in the code instead of using a compile definition. I don't know how to achieve that at all :(

Reply:
GstPlugin struct has a basename prop that should be what we need, but I don't know how to access it. If we get that value it should work with both installed and uninstalled. I'll see if I can get it somehow (maybe you know how to and can spare me the research?)

Reply:
good idea, `g_path_get_basename (gst_plugin_get_filename (plugin))` ?

Reply:
yep it can be achieved with `g_path_get_dirname(gst_plugin_get_filename(plugin)`. The only problem I have now is that I don't see how to get the plugin at all in `gstcefsrc.cc` file (where I need to construct the path), and I can't see any way to send the information from `gstcef.cc` (where I do have the plugin). I can get over that by iterating `gst_registry_get_plugin_list(gst_registry_get())` to get the plugin though. Do you know of a way to get the plugin or pass the path where it's needed, or think using the registry would be fine?

Reply:
I think you want `gst_plugin_feature_get_plugin(GST_PLUGIN_FEATURE (gst_element_factory_find ("cefsrc")));`

Reply:
I arrived to `gst_registry_find_plugin(gst_registry_get(), "cef");` which seams to do the same thing in this case. I'll do a few more tests.

Reply:
yes, that should be equivalent

Reply:
I'll close this PR and open a new one with the other aproach as it solves the issue much better (making the plugin fully relocatable), if that's ok with you :)